driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -1089,7 +1093,12 @@ public void run() {
     }
 
     private static boolean isThriftSupported(VersionNumber cassandraVersion) {
-      return cassandraVersion.compareTo(VersionNumber.parse("4.0")) < 0;
+      // Thrift is removed from some pre-release 4.x versions, make the comparison work for those
+      return cassandraVersion.compareTo(VersionNumber.parse("4.0-a")) < 0;
```

Comment:
See my comment in the other PR, you should be able to use `VersionNumber.nextStable()` for that.